src/WatchDogs.FakeSource/TradeModel.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Bogus;
+
+namespace WatchDogs.FakeSource;
+
+public record TradeModel
+
+    //Guid DealsGuid,
+    //decimal AcountBalance,
+    //TradeAction Action,
+    //DateTimeOffset TimeStamp,
+    //string Currency,
+    //decimal Lot
+
+    //Why Tim want to make this record instead of class??? Ask Mato.
+
+{
+    public Guid DealsGuid { get; set; }
```

Comment:
DealsGuid doesn't make much sense. Guid is simply a random 128 bit identifier. Just name it Id.

Reply:
Done.